TRAMS-API/ODataHelpers/D365ModelHelper.cs

```diff
@@ -150,7 +150,9 @@ private List<PropertyInfo> GetTypeProperties(Type type)
         /// <returns></returns>
         private static bool IsSystemType(PropertyInfo p)
         {
-            return p.PropertyType.FullName.Contains("System.") && !p.PropertyType.FullName.Contains("System.Collections.Generic.List");
+            return p.PropertyType.FullName.Contains("System.") && 
+                  !p.PropertyType.FullName.Contains("System.Collections.Generic.List") ||
+                   p.PropertyType.IsEnum;
```

Comment:
Question:  Curious about this bit and whether some brackets are required here to group the intent.  

Does this read as:

(Type derives from "System" and not from "List") or (Type is an enum)

or

Type derives from "System" and (not from "List" or Type is an enum)

Reply:
well, && has a higher precedence than || so it's kind of like reading 2 x 4 + 6
(Type is system but not a List) or (Type is Enum)
Will add brackets to help with reading